example-get-started-experiments/code/sagemaker/deploy_model.py

```diff
@@ -0,0 +1,46 @@
+import logging
```

Comment:
This is just standard code suggested by sagemaker docs :

https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/using_pytorch.html#deploy-pytorch-models